test-data/unit/check-isinstance.test

```diff
@@ -434,13 +434,13 @@ def foo() -> Union[int, str]: pass
 
 x = foo()
 x = 1
-x = x + 1
```

Comment:
Because of #3122. This test doesn't actually care about that behavior, it simply wanted to see if `x + 1` is allowed. But until #3122 is fixed, the side effect is that `x` changes type in a inconsistent manner, making the test incorrect without fixtures.